### ISSUE TITLE
Different usage of froide

### ISSUE THREAD

**Issue by pdehaye** [stefanw/froide#176]
Hi

I was wondering if you would think it is possible to use Froide for a slightly different purpose than Freedom of Information requests. Instead, I am imagining a usage similar to [this](https://selbstauskunft.net/): a site that helps individuals exercise their access rights with companies to their own personal data. Additionally, it would also help users to see each others' requests, so users know what works and what doesn't, which companies give data easily, etc.

I do not quite understand how Froide works, but am assuming it is sending the FOIA emails on behalf of the users directly, and receiving them as well (right?). This would not do in the context of personal data. Users would ideally need to be able to define a private and a public space to compose requests, and responses need to be kept private by default, with the option for users of making some responses public. 

Indeed, typically a request for such information leads to a long exchange with a corporate lawyer:
 - about what that site/company considers to be personal data, and usually how they are wrong;
 - about how to authenticate the requester.

Both of those are very much worth sharing, but the exchange also requires at some point sending that authentication information (say scan of passport), and eventually receiving in return the data (always unsure what you will find there, so by default this should be private).

I would appreciate if you could give me some evaluation of the difficulty of  using Froide for that purpose.

**Comment by fin** [comment]
so, seems like the initial steps - searching for a recipient, sending a request, would be very similar to how froide handles things. the receiving steps, less so, on a few dimensions:

answers can be added automatically or entered manually. if a request is public, the answer is public. the exception is attachments, since we allow redaction of attachments while doing automatic redaction on text content (blanking names or addresses if necessary).

as you say, This would not do in the context of personal data.

so basically you'd have to take out the receiving stage and replace it with a more workflow based system. this is technically possible, but it's not a straightforward change in the software. as a maintainer, i'd be very hesitant to bring in this functionality because of the added complexity. (but i'm interested in what @stefanw has to say about this).

**Comment by pdehaye** [comment]
This is already very helpful! So attachments in responses are not automatically public. But everything has to go through the central server, correct?
Paul

**Comment by fin** [comment]
yes, froide is intended to be a centralized web service.
